scm-ui/ui-components/src/layout/Footer.tsx

```diff
@@ -89,14 +96,24 @@ const Footer: FC<Props> = ({ me, version, links }) => {
     meSectionTile = <TitleWithIcon title={me.displayName} icon="user-circle" />;
   }
 
+  let meSectionBody = <div />;
+  // Do not show the additional links in footer if user is anonymous
+  if (links?.logout) {
+    meSectionBody = (
+      <>
+        <NavLink to="/me/settings/password" label={t("profile.changePasswordNavLink")} />
```

Comment:
What is the reason for not rendering the password link if me does not have a logout link?

Reply:
If the user has a logout link, it cannot be anonymous. I thought this would be nicer than `me.name !== "_anonymous"`

Reply:
But why do not check if the has a password link? I'm also not sure if is ok, to not render the profile settings extensions point for the anonymous user. Maybe there is a plugin, which has a use case.